tests/e2e/english.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { test, expect } from "@playwright/test"
+
+test("should contain English text", async ({page}) => {
+    await page.goto("/en");
+    expect(await page.title()).toBe("Česko.Digital – Skrz jedničky a nuly měníme Česko k lepšímu");
```

Comment:
Ten test na `<title>` bychom možná mohli vypustit? Případně přesunout do `homepage.spec.ts`?

Reply:
Jsem to tam nechal, protože mě přišlo, že to je ten test, co by měl havarovat (protože ten title je česky).

Reply:
To je spíš havárie na straně toho webu :) Ten `<title>` bychom měli v angličtině předělat, mrknu na to.